cmd/podman/report/format.go

```diff
@@ -0,0 +1,66 @@
+package report
+
+import (
+	"reflect"
+	"strings"
+)
+
+// TableReplacer will remove 'table ' prefix and clean up tabs
+var TableReplacer = strings.NewReplacer(
+	"table ", "",
+	`\t`, "\t",
+	" ", "\t",
```

Comment:
```suggestion
	"\\t ", "\t",
```
Docker formatting will take in a \t in the command line and change it into a tab. I think double backslash escapes the backslash escape, so that this will work?

```
$ sudo docker ps --format "table {{.Names}}\t{{.Status}}\t{{.RunningFor}}\t{{.Image}}\t{{.Ports}}"
NAMES                 STATUS                            CREATED             IMAGE               PORTS
condescending_gould   Up About a minute                 2 minutes ago       nginx:alpine        80/tcp
```

Reply:
@ashley-cui Back quotes in go are raw string literals so \`\t\` will cover your use case. The double quotes are interpreted string literals so `"\t"` is replaced by the tab character.

Unless you are asking for three runes `\ + t + <sp>` to be replaced by tab, that is in your comment and the current replacer will not make that change but I can add it if you need it.

Reply:
ah okay! didn't see the backquotes, LGTM